pkg/rc/replication_controller.go

```diff
@@ -699,7 +742,7 @@ func (rc *replicationController) transferNodes(rcFields fields.RC, ineligible []
 	rc.nodeTransfer.id = nodeTransferID
 	rc.nodeTransferMu.Unlock()
 
-	err = rc.scheduleWithSession(rcFields, newNode)
+	err = rc.scheduleNewNodeForTransfer(rcFields, newNode, current)
```

Comment:
nit: the error message on line 751 still says "Could not schedule _with session_"